foxglove_data_platform/client.py

```diff
@@ -290,6 +290,9 @@ def get_messages(
         """
         Returns a list of tuples of (topic, raw mcap record, decoded message).
 
+        .. deprecated:: 0.13.0
```

Comment:
If we're going to deprecate this, we should create follow-up issues to update the [jupyter example](https://github.com/foxglove/tutorials/tree/main/data-platform/jupyter-notebook) and possibly [associated docs](https://foxglove.dev/blog/analyze-your-robotics-data-with-jupyter-notebooks) (I think the blog post is the only real documentation for this).